### DISCUSSION TITLE
cannot mirror my code in GitHub

### DISCUSSION THREAD

**Discussion by figojoel** [community/community#28159] [Apps, API and Webhooks]
I have a private repo on GitLab that I would love to import in a private repo in GitHub. Whenn i try to import it with `git push --mirror https://github.com/ghuser/repo.git` i receive an error 

![image](https://user-images.githubusercontent.com/63900179/182577052-f771de5a-cf09-49ae-8e6a-64d1f17e73e4.png)

Thanks for your help!!!

**Comment by airtower-luna** [comment]
The message tells you what's wrong: GitHub limits the size of files you can have in a repository, and your repository has a file that's too big. See [About large files on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github) for a description of those limits.

In short, you either need to remove the file from history (possibly move it to LFS), or find some other hosting for your repository. If `.apk` is what I assume it is (an Alpine package) having it in a repository seems at least questionable. :sweat_smile:

**Comment by chienhsiang-hung** [comment]
GitHub only allows for 100 MB files.
try something like this: (adjust it under the comment to make it fit your need)
```
$ git lfs install
$ git lfs track "*.psd"
$ git add .gitattributes

# just commit and push to GitHub as you normally do
$ git add file.psd
$ git commit -m "add design file"
$ git push origin master
```

**Reply by airtower-luna** [reply]
That alone helps only if the large file isn't already part of the history (which it is in this case). The push will still fail unless the large file is removed from the history first.

**Reply by figojoel** [reply]
yes you had right @airtower-luna... the file was a part of the history. I deleted it with `git filter branch...`. -methode. It's working now. 
Thanks for your contributions.
